### ISSUE TITLE
Mission Windurst 8-2 Jester Who'd Be King - NMs fail to spawn at G-8 Cracked Wall

### ISSUE THREAD

**Issue by Kalifriki** [DerpyProjectGroup/topaz#1526]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: Rebirth

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
https://ffxiclopedia.fandom.com/wiki/The_Jester_Who%27d_Be_King

Mission interactions are OK until attempting to spawn NMs at G-8 Cracked Wall in [Outer Horutoto Ruins] at [F-4] in [West Sarutabaruta]. No Cardians spawn, no dialog. Waiting 5-30mins, relogging, no change. No change regardless of effects of Party or Solo status, with/without Pet, and with or without Sneak.
![Bug_Mission_8-2-CrackedWall_Progress](https://user-images.githubusercontent.com/46100825/162602349-491835df-efa5-4258-80ce-976ea4963267.png)
![Bug_Mission_8-2-CrackedWallG8](https://user-images.githubusercontent.com/46100825/162602351-8ab1d506-9231-4f4a-ac80-8447d72f42e1.png)

**Comment by RAIST5150** [comment]
Have you verified you got the ring key items from the Optistery, Davoi, Fei'yin, and Heavens Tower?

These must be run in specific order to advance a tracked value as well.  If that value is not advanced properly,  it will stall progression.  In the past we have seen such issues resolved by going through the dialogue or cut scenes again.

**Comment by Kalifriki** [comment]
Yes, all KI rings are accounted for in order of wiki walkthrough. Dialog between Apuruu and Kupipi trigger as intended. The Cracked Wall at G-8 will not populate NMs, the mission status was not set correctly...either from glitch or a lack of update from conversation with NPC prior to interaction with Cracked Wall.
